App.js

```diff
@@ -1,8 +1,8 @@
-import * as Sentry from '@sentry/react-native';
 import AsyncStorage from '@react-native-async-storage/async-storage';
+import * as Sentry from '@sentry/react-native';
 import * as Font from 'expo-font';
```

Comment:
`import { useFonts } from 'expo-font';` from the docu

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/bb2bdc60ccd5f64a66494d86ad10a033f3cf4e4b